dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -959,9 +963,9 @@ def generate_teacher_student(name, teacher_authorized)
   email, password = generate_user(name)
 
   steps %Q{
-    Given I am on "http://studio.code.org/reset_session"
-    Given I am on "http://studio.code.org/users/sign_up?user%5Buser_type%5D=teacher"
+    Given I am on "http://studio.code.org/users/sign_up"
     And I wait to see "#user_name"
+    And I select the "Teacher" option in dropdown "user_user_type"
```

Comment:
I change this create a teacher step to match the create a student step because occasionally when running these new tests locally they would hang waiting for the teacher's age. 
<img width="702" alt="screen shot 2018-03-28 at 3 56 33 pm" src="https://user-images.githubusercontent.com/12300669/38062492-02f0f9fc-32a9-11e8-94a6-b876632b15d2.png">
With this revision, the tests passed multiple times and more quickly.

Reply:
Nice find, nice fix!